docker/docker-compose.yaml

```diff
@@ -2,24 +2,52 @@ version: '3.7'
 
 volumes:
   textile-fc-data:
+  textile-fc-ipfs:
+  textile-fc-lotus:
 
 services:
-  textile-fc:
-    network_mode: host
+  
```

Comment:
Since now this _docker-compose_ file also includes the Lotus node, we don't have the problem of _fil-tools_ container needing to access a host port (where lotus was running). This means we don't need fancy _network-modes_ since _fil-tools_ now communicates to another container in the same docker-compose environment.

This is good since _network_mode=host_ was only supported in Linux... so now this can be run in Darwin too.